lang/ff7tk_fr.ts

```diff
@@ -337,7 +337,7 @@
     </message>
     <message>
         <source>Young Cloud</source>
-        <translation type="unfinished">Jeune Clad</translation>
+        <translation>Jeune Cloud</translation>
```

Comment:
I thought in the french game it was Clad ?

Reply:
Only on the PlayStation 1 release, it was Cmoud on any other versions